Allow dbf 5.0+ as dependency

ilvez opened the pull request:
dbf released [version 5.0.0](https://github.com/infused/dbf/blob/master/CHANGELOG.md#500) (and 5.0.1), which offer support for UTF-8 encoded columns.

I made dbf version required more loose, since 5.0 does not break any compatibility. I'm not library developer, so I don't know whether it's better to keep dependency versions more restricted or looser. In this case dbf 5.0 does not break compatibility in any way.

ilvez committed:
chore: allow dbf 5.0+ as dependency

BuonOmo commented:
I would limit to 5.X though, 6 may break something in the future. WDYT?

Besides that, I'm ok with it seeing the changelog. @keithdoggett opinion?

ilvez committed:
chore: update changelog/history

ilvez commented:
> I would limit to 5.X though, 6 may break something in the future. WDYT?

I updated to not include major after 5.

> Besides that, I'm ok with it seeing the changelog. @keithdoggett opinion?

Thanks, I was skimming through files and didn't see CHANGELOG. Updated History.md now.

ilvez commented:
Can I help somehow to get this merged?

BuonOmo commented:
@ilvez I don't think. @keithdoggett should review it and we'll be set :)

keithdoggett reviewed (approved):
LGTM sorry about the delay
